lib/elixir_google_scraper/oauth_access_grants/oauth_access_grant.ex

```diff
@@ -0,0 +1,10 @@
+defmodule ElixirGoogleScraper.OauthAccessGrants.OauthAccessGrant do
```

Comment:
This schema file(also `oauth_access_token.ex`, `oauth_application.ex`) is auto-generated by the library. I tried to move it `schemas` folder as we do for schemas. But it does not work as the `ex_oauth2_provider` library's internal code depends on this path to access the schema. That's why I keep it as it is.